src/asm/gamepad.s

```diff
@@ -171,6 +174,48 @@ SoftReset:
 .assert * <= $cbd3
 .endif
 
+.ifdef _RANDOM_WILD_WARP
+.org $cbd3
+  jsr LoopWildWarpIndex
+  ldy $0780
+  lda WildWarpLocations,y
+  sta $6c ; CurrentLocation
+  lda #$00
+  sta $6d ; CurrentEntrance
+  inc $0780
+  lda #$01 ; GAME_MODE_CHANGE_LOCATION
+  sta GameMode
+  pla
+  pla
+  rts
+.assert * = $cbec
+  
+.import wildWarpCount
+  
+.reloc
+LoopWildWarpIndex:
+  ; happening before warping, so $0780 should contain index we want to go to
+  lda $0780
+  cmp #$ff ; indicates we looped from 1 -> count
+  bne +
+    lda #wildWarpCount
```

Comment:
how about `clc; adc #wildWarpCount`?  Then you can skip the `bne ++` because it'll basically be the positive version of the same number.